webrtc.html

```diff
@@ -13326,6 +13333,12 @@ <h2>
                       negotiation-needed flag =] for <var>connection</var>.
                     </p>
                   </li>
+                  <li>
+                    <p>
+                      [=list/Append=] <var>channel</var> to
+                      <var>connection</var>.{{RTCPeerConnection/[[DataChannels]]}}.
```

Comment:
It seems this should be done for both createDataChannel code and ondatachannel code paths.
We could add it to https://w3c.github.io/webrtc-pc/#announcing-a-data-channel-instance for instance.